service/worker/migration/workflow.go

```diff
@@ -62,10 +64,12 @@ const (
 type (
 	ForceReplicationParams struct {
 		Namespace               string
-		SkipAfterTime           time.Time // skip workflows that are updated after this time
-		ConcurrentActivityCount int32
-		RemoteCluster           string // remote cluster name
+		Query                   string // query to list workflows for force replicaiton
+		ConcurrentActivityCount int
+		MaxActivityCount        int    // max activities before continue as new
 		RpsPerActivity          int    // RPS per each activity
+		ListWorkflowsPageSize   int    // needed for testing purpose, default to 1K
```

Comment:
I'm confused, does this mean you don't use paging in production?  I don't see that in the code.